lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Kevin Breit (@kbreit) <kevin.breit@kevinbreit.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: meraki_network
+short_description: Manage networks in the Meraki cloud
+version_added: "2.6"
+description:
+- Allows for creation, management, and visibility into networks within Meraki.
+
+notes:
+- More information about the Meraki API can be found at U(https://dashboard.meraki.com/api_docs).
+- Some of the options are likely only used for developers within Meraki.
+
+options:
+    auth_key:
+        description:
+        - Authentication key provided by the dashboard. Required if environmental variable MERAKI_KEY is not set.
+    state:
+        description:
+        - Create or modify an organization.
+        choices: ['present', 'query', 'absent']
```

Comment:
So, my personal view here is that if you list items, and the order has no real meaning, it's better to order them alphabetically.

Reply:
There's also no need to quote the values here.
Another rule I abide to is to not quote strings that not need quoting in YAML.
Otherwise people will not understand the YAML rules to quoting, and start quoting everything always.